tests/fixtures/src/MockSiteSectionProvider.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\Fixtures;
+
+use Vanilla\Contracts\Site\SiteSectionInterface;
+use Vanilla\Contracts\Site\SiteSectionProviderInterface;
+
+/**
+ * Mock site-section-provider.
+ */
+class MockSiteSectionProvider implements SiteSectionProviderInterface {
+
+    /**
+     * @var array $siteSections
+     */
+    private $siteSections = [];
+
+    /**
+     * MockSiteSectionProvider constructor.
+     */
+    public function __construct() {
+        $this->siteSections = self::createSiteSections();
+    }
+
+
+    /**
+     * @inheritdoc
+     */
+    public function getAll(): array {
+        return $this->siteSections;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getByID(int $id): ?SiteSectionInterface {
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getByBasePath(string $basePath): ?SiteSectionInterface {
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getForLocale(string $localeKey): array {
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getCurrentSiteSection(): SiteSectionInterface {
+    }
+
+    /**
+     * Create site-sections to a section group.
+     * @return array
+     */
+    public static function createSiteSections() {
+        $locales = ["en", "fr", "es", "ru"];
+        $siteSections = [];
+
+        foreach ($locales as $locale) {
+            $siteSectionPath = $locale.'/';
```

Comment:
Is the trailing slash here normal? Most things in Vanilla making URLs don't have a trailing slash to start.

Reply:
in the SubcommunitySiteSection a `/` is appended to the folder

https://github.com/vanilla/multisite/blob/b21973009046574e34ff12f7f3b3e2cb8442abd8/plugins/subcommunities/models/SubcommunitySiteSection.php#L49